src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -160,16 +196,26 @@ public static function register($mode = 0)
                     $colorize = function ($str) { return $str; };
                 }
                 if ($currErrorHandler !== $deprecationHandler) {
+                    print_r(error_get_last());
```

Comment:
the link between `error_get_last()` and the error handler change - maybe I'm wrong - it's still unrelated so please open another PR if you want to investigate the idea